src/test/java/org/opensearch/neuralsearch/search/query/HybridCollectorManagerTests.java

```diff
@@ -498,7 +498,7 @@ public void testReduce_whenMatchedDocsAndSortingIsApplied_thenSuccessful() {
     }
 
     @SneakyThrows
-    public void testReduceWithConcurrentSegmentSearch_whenMultipleCollectorsMatchedDocs_thenSuccessful() {
+    public void testReduceWithConcurrentSegmentSearch_whenMultipleCollectorsMatchedDocs_thenSuccessful() throws IOException {
```

Comment:
throws is not required, @SneakyThrows should take care of all this

Reply:
+1